ITE/4/README.adoc

```diff
@@ -234,27 +237,27 @@ properties. For example, an implementation that is capable of hashing GitHub
 entities may choose to use a subsection of the contents obtained using the
 official API, which returns JSON representations containing different properties
 of the entities. It is important for these representations to be consistent for
-a particular entity, and to acknowledge the different
-sources of non-determinism highlighted in the Appendix. It is also necessary
-for this method to handle any missing resources safely. That is, if a resource does not exist
-at the location pointed to by a URI, this method must fail appropriately.
-
-Being able to get some hashable representation of abstract entities is also needed
-for in-toto inspections to verify contents. There is no specification mandating
-any specific inspections. Yet, these abstract resources still
-need to be resolvable so that they can be inspected if the supply chain owner so decides to.
-Therefore,
-it should be possible for implementations to use `get_hashable_representation` in scenarios other
-than just recording the hashes of an artifact.
-
-It is quite likely that any given generic URI specified could point to a collection of
-abstract entities. In the present framework where all artifacts are part of the
-filesystem, this is akin to using the path of a directory to record all the
-contents of the directory as the actual artifacts of the step.
+a particular entity, and to acknowledge the different sources of non-determinism
+highlighted in the Appendix. It is also necessary for this method to handle any
+missing resources safely. That is, if a resource does not exist at the location
+pointed to by a URI, this method must fail appropriately.
+
+Being able to get some hashable representation of abstract entities is also
+needed for in-toto inspections to verify contents. There is no specification
+mandating any specific inspections. Yet, these abstract resources still need to
+be resolvable so that they can be inspected if the supply chain owner so decides
+to. Therefore, it should be possible for implementations to use
```

Comment:
How does this sentence follow from the previous one? Not clear to me, at least.

Reply:
I think I've clarified this one now. Basically, we're stating that `get_hashable_representation` should be callable on demand, for things like inspections.